app.rb

```diff
@@ -1,163 +1,20 @@
-    choice = gets.chomp.to_i
-    case choice
-    when 1
-      register_student
-    when 2
-      register_teacher
-    else
-      puts 'Incorect choice, choose a number between 1-2'
-      create_person
-    end
-  end
-
-  def register_student
-    print 'Age: '
-    age = gets.chomp.to_i
-
-    print 'Name: '
-    name = gets.chomp
-
-    print 'Classroom: (C1,C2): '
-    classroom = gets.chomp
-
-    has_permission = permission?
-
-    student = Student.new(classroom, age, name: name, parent_permission: has_permission)
-    @people << student unless @people.include?(student)
-
-    puts "\n Student '#{name}' aged '#{age}' with the classroom '#{classroom}' created Successfully 👍"
-  end
-
-  def register_teacher
-    print 'Age: '
-    age = gets.chomp.to_i
-
-    print 'Name: '
-    name = gets.chomp
-
-    print 'Specialization: '
-    specialization = gets.chomp
-
-    teacher = Teacher.new(specialization, age, name: name)
-    @people << teacher unless @people.include?(teacher)
-
-    puts "\n Teacher '#{name}' aged '#{age}' with the specialization '#{specialization}' created Successfully 👍"
-  end
-
-  def permission?
-    print 'Has parent permission [Y/N]: '
-    permission = gets.chomp
-
-    case permission
-    when 'n', 'N'
-      false
-    when 'y', 'Y'
-      true
-    else
-      puts "\nIncorect choice, please press 'n'/'N' for false and 'y'/'N' for true"
-    end
-  end
-
-  # Books
-  def create_book
-    print 'Title: '
-    title = gets.chomp
-
-    print 'Author: '
-    author = gets.chomp
-
-    book = Book.new(title, author)
-    @books << book unless @books.include?(book)
-
-    puts "\n Book is successfully created 📚"
-  end
-
-  # Rentals
-  def list_rentals_by_id
-    print "Enter a person's ID: "
-    person_id = gets.chomp.to_i
-    if !@people.find { |person| person.id == person_id }
-      puts "No rental found with ID: #{person_id}"
-    elsif @rentals.empty?
-      puts 'The rental list is empty'
-    else
-      puts "\nRentals #{@rentals.count}:\n"
-      @rentals.select do |rental|
-        if rental.person.id == person_id
-          puts "Date: #{rental.date}, Book: '#{rental.book.title}' by #{rental.book.author}"
-        end
-      end
-    end
-  end
-
-  def create_rental
-    print "Select a book from the following list by number:\n"
-    list_all_books
-    prompt_user if @books.empty?
-    book_index = gets.chomp.to_i - 1
-
-    print "Select a person from the following list by number:\n"
-    list_all_people
-    prompt_user if @people.empty?
-    person_index = gets.chomp.to_i - 1
-
-    print 'Date [yyyy/mm/dd]: '
-    date = gets.chomp
-
-    rental = Rental.new(date, @people[person_index], @books[book_index])
-    @rentals << rental unless @rentals.include?(rental)
-
-    puts "\nRental created successfully 👍 \n\n"
-  end
-
-  def quit_app
-    puts 'Thank you for using this App! 🙏'
-    exit
-  end
+  include ListBooks
+  include ListPerson
```

Comment:
- [ ] I think those are not the right modules name ad I can see you refactor it to be ShowBook and ShowPeople but you didn't change them here so I recommend to change them to make your app works properly 🔥💯✅


![image](https://user-images.githubusercontent.com/58553711/211437239-bcc694e0-7c77-4fd4-9633-3465dd0bcb1e.png)